test/models/embrake.jl

```diff
@@ -1,116 +0,0 @@
-set_rtol(Float64, 1e-12)
```

Comment:
removing this file seems unrelated

Reply:
better to resolve after it's addressed so we don't forget

Reply:
oh, i see now -- you want to delete this file because it's a duplicate with `EMBrake.jl`  and we only load the latter in the tests?

Reply:
I had my doubts with that...